src/cli.rs

```diff
@@ -252,6 +259,90 @@ pub fn run() -> Result<()> {
     Ok(())
 }
 
+fn validate_config(conf: &Config) -> Result<()> {
```

Comment:
If you wouldn't mind adding a doc comment to this function, that would be great as well!